src/components/interfaces/HMI_API.xml

```diff
@@ -2356,8 +2356,8 @@
       <param name="fileName" type="String" maxlength="255" minlength="1" mandatory="true">
           <description>The path to file.</description>
       </param>
-      <param name="appID" type="Integer" mandatory="false">
-        <description>ID of application that requested this RPC.</description>
+      <param name="appID" type="Integer" mandatory="true">
```

Comment:
@orhan-mehmedov Please change version according to https://adc.luxoft.com/confluence/display/APPLINK/API+xml+versioning+approach